quickfixj-core/src/test/java/quickfix/SessionTest.java

```diff
@@ -101,7 +101,7 @@ public void testDisposalOfFileResources() throws Exception {
                 new DefaultMessageFactory(), 30, false, 30, UtcTimestampPrecision.MILLIS, true, false,
                 false, false, false, false, true, false, 1.5, null, true,
                 new int[] { 5 }, false, false, false, false, true, false, true, false,
-                null, true, 0, false, false, true, new ArrayList<>())) {
+                null, true, 0, false, false, true, new ArrayList<>(), 1.4)) {
```

Comment:
Instead of using this magic number, what about using `Session.DEFAULT_HEARTBEAT_TIMEOUT_MULTIPLIER` in this test?

Reply:
Changed in the whole file.